File: apps/mobile/lib/core/config/env.dart
class ENV {
  static const String appEnv = String.fromEnvironment('APP_ENV', defaultValue: 'development');
  static const bool useMockApi = bool.fromEnvironment('USE_MOCK_API', defaultValue: true);
  static const String apiBaseUrl = String.fromEnvironment('API_BASE_URL', defaultValue: 'http://localhost:4000/api/v1');
  static const String aiGatewayUrl = String.fromEnvironment('AI_GATEWAY_URL', defaultValue: 'http://localhost:4001/ai/v1');
}

File: apps/admin/src/config/env.ts
/**
 * Typed environment configuration module for Shikkhok-AI Admin Console.
 * Centralizes all environment variable access.
 */
export const ENV = {
	apiBaseUrl: process.env.NEXT_PUBLIC_API_BASE_URL?.replace(/\/+$/, '') || 'http://localhost:3000/api/v1',
	appName: 'Shikkhok-AI Admin Console',
	isProduction: process.env.NODE_ENV === 'production',
	isDevelopment: process.env.NODE_ENV !== 'production'
} as const;

export type Environment = typeof ENV;
